copier/errors.py

```diff
@@ -1,7 +1,11 @@
 """Custom exceptions used by Copier."""
 
+from __future__ import annotations
```

Comment:
I'd prefer removing this line and reverting the typing changes accordingly, and address this change in a separate PR to avoid mixing concerns here.